### ISSUE TITLE
Improve Akashjs validation

### ISSUE THREAD

**Issue by baktun14** [akash-network/console#133]
- Move custom validation done in Cloudmos for the SDL parsing to Akashjs
- Properly show the error message in Cloudmos when the validation throws in akashjs
- Improve the logic between between Cloudmos and the SDL parsing from Akashjs

**Comment by ygrishajev** [comment]
@baktun14 why moving it to akashjs? is it needed anywhere else besides cloudmos?

**Comment by baktun14** [comment]
Initially the sdl parsing was done in Cloudmos for Cloudmos and akashjs implemented the same logic so we migrated to use akashjs's implementation. However, there's some validation done to make sure the SDL is valid [that we implemented in Cloudmos](https://github.com/akash-network/cloudmos/blob/main/deploy-web/src/utils/deploymentData/v1beta3.ts#L12-L213) that was [partially implemented in akashjs](https://github.com/akash-network/akashjs/blob/main/src/sdl/index.ts#L67-L78).

So now it's just a matter of moving all the validation logic to the same place, in AkashJS.

For context, we implemented the validation based on the [validation done on the nodes in go code](https://github.com/akash-network/node/blob/76204ef59cfdd741c10cba220240056029ac8b7c/sdl/v2.go#L255). So we can validate the SDL dynamically in the client without getting an error along the way when creating the transactions to create a deployment or to send a manifest.
